cms/templates/js/partition-group-details.underscore

```diff
@@ -23,25 +23,29 @@
    </ol>
 
     <ul class="actions group-configuration-actions">
-        <li class="action action-edit">
-            <button class="edit"><span class="icon fa fa-pencil" aria-hidden="true"></span> <%- gettext("Edit") %></button>
-        </li>
-        <% if (_.isEmpty(usage)) { %>
-            <li class="action action-delete wrapper-delete-button" data-tooltip="<%- gettext('Delete') %>">
-                <button class="delete action-icon" title="<%- gettext('Delete') %>"><span class="icon fa fa-trash-o" aria-hidden="true"></span></button>
-            </li>
-        <% } else { %>
-            <li class="action action-delete wrapper-delete-button" data-tooltip="<%- gettext('Cannot delete when in use by a unit') %>">
-                <button class="delete action-icon is-disabled" disabled="disabled" title="<%- gettext('Delete') %>"><span class="icon fa fa-trash-o" aria-hidden="true"></span></button>
+        <% if (!restrictEditing) { %>
+            <li class="action action-edit">
+                <button class="edit"><span class="icon fa fa-pencil" aria-hidden="true"></span> <%- gettext("Edit") %></button>
             </li>
+            <% if (_.isEmpty(usage)) { %>
+                <li class="action action-delete wrapper-delete-button" data-tooltip="<%- gettext('Delete') %>">
+                    <button class="delete action-icon" title="<%- gettext('Delete') %>"><span class="icon fa fa-trash-o" aria-hidden="true"></span></button>
+                </li>
+            <% } else { %>
+                <li class="action action-delete wrapper-delete-button" data-tooltip="<%- gettext('Cannot delete when in use by a unit') %>">
+                    <button class="delete action-icon is-disabled" disabled="disabled" title="<%- gettext('Delete') %>"><span class="icon fa fa-trash-o" aria-hidden="true"></span></button>
+                </li>
+            <% } %>
         <% } %>
     </ul>
 </div>
 
 <% if (showContentGroupUsages) { %>
   <div class="collection-references wrapper-group-configuration-usages">
     <% if (!_.isEmpty(usage)) { %>
-      <h4 class="intro group-configuration-usage-text"><%- gettext('This content group is used in:') %></h4>
+      <h4 class="intro group-configuration-usage-text">
+        <%= gettext('This group controls visibility of:') %>
```

Comment:
Should be <%- instead of <%=. The latter doesn't do escaping, and should only be used with HTML (which has already been run through HtmlUtils).